src/hotspot/share/runtime/sharedRuntime.cpp

```diff
@@ -237,12 +238,11 @@ JRT_LEAF(jfloat, SharedRuntime::frem(jfloat  x, jfloat  y))
 #ifdef _WIN64
   // 64-bit Windows on amd64 returns the wrong values for
   // infinity operands.
-  union { jfloat f; juint i; } xbits, ybits;
-  xbits.f = x;
-  ybits.f = y;
+  juint xbits_i = PrimitiveConversions::cast<juint>(x);
+  juint ybits_i = PrimitiveConversions::cast<juint>(y);
```

Comment:
The "_i" suffix here doesn't have any meaning anymore.  Just "xbits" and "ybits" would be better.
Similarly in below in "drem".